<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCaughtFish</class>
 <widget class="QDialog" name="DialogCaughtFish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="stashFishButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>721</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Stash Fish</string>
   </property>
  </widget>
  <widget class="QLabel" name="fishImageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
    </font>
   </property>
   <property name="text">
    <string>Fish Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fishNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>MagiCarp</string>
   </property>
  </widget>
  <widget class="QLabel" name="staticDescriptionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>376</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
    </font>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>411</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>- Magicarp is a fish pokemon from Pokemon! More information goes here...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="staticQuickFactsLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
    </font>
   </property>
   <property name="text">
    <string>Quick Facts!</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="staticQuickFactsScrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>419</width>
      <height>139</height>
     </rect>
    </property>
    <widget class="QLabel" name="fishFactOneLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string> - Magicarp evolves into Gyarados!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="fishFactTwoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string> - Abilities include Swift and Swim!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="fishFactThreeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string> - Eats 'Plump Beans' or 'Bean Cake.'</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
